echo-pubsub-google/src/main/resources/gcb.jinja

```diff
@@ -6,5 +6,14 @@
     "name": "{{ image.name }}",
     "version": "{{ image.digest }}",
     "type": "docker/image"
-  } {% if not loop.last %} , {% endif %} {% endfor %}
+  } {% if (not loop.last) or ((artifacts.objects.paths | length) > 1) %} , {% endif %} {% endfor %}
```

Comment:
Why `> 1` instead of `>= 1`? I would think that if there are any items we need a comma

Reply:
Oops; fixed.